cargo-cyclonedx/src/cli.rs

```diff
@@ -37,6 +38,29 @@ pub struct Args {
     #[clap(long = "quiet", short = 'q')]
     pub quiet: bool,
 
+    // The feature selection flags are not mutually exclusive in Cargo,
+    // so we keep the same behavior here too.
+    /// Activate all available features
+    #[clap(long = "all-features")]
+    pub all_features: bool,
+
+    /// Do not activate the `default` feature
+    #[clap(long = "no-default-features")]
+    pub no_default_features: bool,
+
+    /// Space or comma separated list of features to activate
+    #[clap(long = "features", short = 'F')]
+    pub features: Vec<String>,
+
+    /// The target to generate the SBOM for, e.g. x86_64-unknown-linux-gnu
+    #[clap(long = "target")]
+    pub target: Option<String>,
+
+    /// Include the dependencies from all possible target platforms in the SBOM
+    #[clap(long = "all-targets")]
+    #[clap(conflicts_with = "target")]
+    pub all_targets: bool,
```

Comment:
I _think_ I like it better your way but `cargo tree` uses `--target all` for this.

Reply:
I've changed it to the `cargo tree` way for consistency. I'm not a fan of it, but following a convention is usually preferable.

Reply:
Fine with me as well. A comment would be useful why we make this decision for later generations to understand.